src/System.Runtime.InteropServices/ref/System.Runtime.InteropServices.cs

```diff
@@ -204,6 +204,10 @@ public sealed partial class ComAliasNameAttribute : System.Attribute
         public ComAliasNameAttribute(string alias) { }
         public string Value { get { throw null; } }
     }
+    public static partial class CollectionsMarshal
```

Comment:
This seems like alphabetically it should be before `ComAliasNameAttribute`, no?

Reply:
Done